### PR TITLE
New beginners guide to firewalld

### PR DESCRIPTION
**Pull request by sspencerwire**
* tested rules using an LXD container
* did an editing pass checking word usage, commas, and general feel
* would really, really, really like some input on this

#### Author checklist (Completed by original Author)
- [x] Contribution a good fit for the Rocky project? Title and Author MetaTags inserted ?
- [ ] Is this a non-English contribution? 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Check that document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Basic Editorial Review)
- [x] 4th Pass (Detailed Editorial Review and Peer Review)
- [x] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by sspencerwire** [comment]
I would love someone to review this with a technical eye, particularly if you have experience using `firewalld`. Please add your comment and suggestions!

**Comment by EzequielBruni** [comment]
Alright, I gave it an editing pass, adding some bits I thought might be useful. I only have one overall piece of feedback: this isn't strictly a beginner's guide to firewalld, so much as it is an `iptables` user's guide to firewalld. Maybe we should change the title to reflect that?

Later on, I could take a lot of what's here and give it the absolute newbie treatment if y'all think that would be good.

**Comment by alemorvan** [comment]
Could we add a paragraph with instructions on how to check firewall state ?

Something like this:

Remember to check the status of your firewall with either :

the command `firewall-cmd`:

```
$ firewall-cmd --state
running
```

the `systemctl` command:

```
$ systemctl status firewalld
```

**Comment by alemorvan** [comment]
I like the Common firewall-cmd Commands table 👍

File: docs/guides/security/firewalld.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+| trusted       | all network connections are accepted                                                                              |
+
+!!! Note
+
+    `firewall-cmd` is the command line program for managing the `firewalld` daemon.
+
+To list existing zones on your system, type:
+
+`firewall-cmd --get-zones`
+
```
Comment (alemorvan) [inline docs/guides/security/firewalld.md:54]:
```suggestion
!!! Warning

    Remember to check the status of your firewall, if the `firewalld-cmd` returns you an error, with either:

    the command firewall-cmd:

    ```
    $ firewall-cmd --state
    running
    ```

    the systemctl command:

    ```
    $ systemctl status firewalld
    ```

```

File: docs/guides/security/firewalld.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+|`firewall-cmd --list-all-zones`      | similar to `firewall-cmd --list-all --zone=[zone]` except it lists *all* of the zones and their contents. |
+|`firewall-cmd --get-default-zone`    | shows the default zone, which is "public" unless it has been changed.                                     |
+|`firewall-cmd --list-services --zone=[zone]`| shows all of the services enabled for the zone.                                                    |
+|`firewall-cmd --list-ports --zone=[zone]`| shows all ports open on the zone.                                                                     |
+|`firewall-cmd --get-active-zones`    | shows the zones that are active on the system, their active interfaces, services, and ports.              |
+|`firewall-cmd --get-services`        | shows all available services possible for use.                                                            |
+|`firewall-cmd --runtime-to-permanent`| if you have entered many rules without the --permanent option, do this before reloading.                  |
+
+There are a great many `firewall-cmd` options not covered here, but this gives you the most used commands.
+
```
Comment (alemorvan) [inline docs/guides/security/firewalld.md:409]: ## Testing connections

System administrators are very fond of the `telnet` command to perform their tests and verify that a port is open and accessible.

Today, the recommended command is `nc` (netcat) which you can use as follows:

```
nc -zv 192.168.1.10 22
```

The `-u` option is use to test udp port by the same way.

**Pull request merged**